src/Backend/Modules/Pages/Installer/Data/locale.xml

```diff
@@ -641,10 +641,9 @@
         <translation language="fr"><![CDATA[Ajouter des tags ici.]]></translation>
       </item>
       <item type="message" name="MovePagesNotPossible">
-        <translation language="nl"><![CDATA[Het verplaatsen van pagina's is niet mogelijk op touch-apparaten.]]></translation>
-        <translation language="en"><![CDATA[Moving pages on touch devices is not possible.]]></translation>
-        <translation language="it"><![CDATA[Spostare pagine su dispositivi touch non è possibile.]]></translation>
-        <translation language="de"><![CDATA[Verschieben von Seiten auf Touch-Geräten ist nicht möglich.]]></translation>
+        <translation language="nl"><![CDATA[Het verplaatsen van pagina's is niet mogelijk hier op touch-apparaten. Ga naar de tab instellingen bij het bewerken van een pagina om de pagina te verplaatsen.]]></translation>
```

Comment:
=> Het verplaatsen van pagina's is niet mogelijk op touch-apparaten. Ga naar ...

Reply:
fixed in cf66f51bf